RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -697,3 +697,120 @@ boolean LX_meatMaid()
 
 	return false;
 }
+
+string LX_getDesiredWorkshed(){
+	string currentWorkshed = get_property("auto_workshed").to_lower_case();
+	//return the actual item name in case a shorthand is used
+	switch(currentWorkshed)
+	{
+		case "model train set":
+		case "train":
+			return "model train set";

```

Comment:
with the return type change, update these returns to something like `return $item[model train set];`